composer.json

```
@@ -120,5 +120,7 @@
         "test-with-postgres": [
             "APP_ENV=test_with_postgres vendor/bin/phpunit --colors=always"
         ]
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```

I'm not sure if prefer stable should stay, but we may leave it for now
```suggestion
    },
    "prefer-stable": true
```